source/modules/MEI.shared.xml

```diff
@@ -6973,55 +6939,23 @@
           <rng:ref name="data.HORIZONTALALIGNMENT"/>
         </datatype>
       </attDef>
+      <attDef ident="func" usage="opt">
+        <desc xml:lang="en">Records the function (i.e., placement) of the page header.</desc>
+        <datatype>
+          <rng:ref name="data.PGFORMFUNC"/>
```

Comment:
We went back and forth with different ideas. `PGFUNC` might seem clearer but can also be misleading because it is not about defining the function of the page.

Reply:
Sorry, I am now understanding the question. I will fix it.

Reply:
Sorry, should have been clearer in my question that the newly added datatype PGFUNC was addressed as PGFORMFUNC here. Thanks for the fix.